code/game/objects/items/grenades/atmos_grenades.dm

```diff
@@ -63,13 +63,13 @@
 			continue
 		var/distance_from_center = max(get_dist(turf_loc, loc), 1)
 		var/turf/open/floor_loc = turf_loc
-		if(floor_loc.air.temperature > 260 && floor_loc.air.temperature < 370)
+		if(floor_loc.air.return_temperature() > 260 && floor_loc.air.return_temperature() < 370)
```

Comment:
for two calls like this it's not a huge deal since it's just literally getting the value, only real cost would be proc overhead and it's not very hot of an area.
This is something to be considered for other places, a lot of the work was very much bulk work with functionality first in mind.